### DISCUSSION TITLE
First Release

### DISCUSSION THREAD

**Discussion by ghost** [IrisShaders/Iris#378] [General]
Devs: Finally, after 8 months of arduous work, we have finished our mod for Minecraft 1.16.5!
1.17: I'm about to end the whole team's career.

**Comment by DragonEggBedrockBreaking** [comment]
This mod has a 1.17 branch

**Reply by Mr-Waterscout** [reply]
The joke is that 1.17 changed a lot of internals making the transition to 1.17 more than just changing the version needed, but also updating the mod to work with the new opengl version and the new java version.
